server/bin/pbench-sync-satellite.sh

```diff
@@ -266,6 +266,13 @@ if [ -s ${state_change_log} ] ;then
     fi
 fi
 
+if [ $nerrs == 0 ] ; then
+    for entry in "${logdir_for_remote}"/*
```

Comment:
Typically, we get the list of files like this and store it in a file, and then read from the file.  Can you look at other examples in the code that follow that pattern?

Reply:
Sure, will look into it

Reply:
@portante May I know why we are copying them first and then deleting instead of deleting the directly?

Reply:
@riya-17, basically line 270 in bash causes the shell to expand all the files that match the pattern, `${logdir_for_remote}/*`, which can potentially be very large.  The shell has a limit to the number of characters in the expansion.  Also, to aid debugging, writing the list of items to be deleted can help us debug the behaviors live.

Reply:
Ok, got it. Thank you